contrib/python/podman/client.py

```diff
@@ -20,37 +133,26 @@ class Client(object):
         >>> import podman
         >>> c = podman.Client()
         >>> c.system.versions
-    """
 
-    # TODO: Port to contextlib.AbstractContextManager once
-    #   Python >=3.6 required
+    Example remote podman:
```

Comment:
Don't you need three double qoutes (""") before this comment block?

Reply:
@TomSweeneyRedHat The docstring is the whole block.  Both examples are included.